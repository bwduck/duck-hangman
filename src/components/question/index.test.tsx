import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import Question from './index';

describe('<Question/>', () => {
  it('renders and matches the snapshot', () => {
    const wrapper = shallow(<Question phrase="this is a test" />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
