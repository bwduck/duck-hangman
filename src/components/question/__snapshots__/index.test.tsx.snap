// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Question/> renders and matches the snapshot 1`] = `
<div
  className="c-question"
>
  <span>
    this is a test
  </span>
</div>
`;
